core/integration/storagetest/batch.go

```diff
@@ -31,24 +31,25 @@ func RunBatchStorageTests(t *testing.T, storageFactory BatchStorageFactory) {
 		"TestReadBatch":  b.TestReadBatch,
 		"TestHighestRev": b.TestHighestRev,
 	} {
-		ms := storageFactory(ctx, t)
-		t.Run(name, func(t *testing.T) { f(ctx, t, ms) })
+		t.Run(name, func(t *testing.T) { f(ctx, t, factory) })
 	}
 }
 
 // BatchTests is a suite of tests to run against
 type BatchTests struct{}
 
-func (*BatchTests) TestNotFound(ctx context.Context, t *testing.T, b Batcher) {
+func (*BatchTests) TestNotFound(ctx context.Context, t *testing.T, f BatchStorageFactory) {
+	b := f(ctx, t, "testnotfounddir")
 	_, err := b.ReadBatch(ctx, "nodir", 0)
```

Comment:
For consistency can "nodir" be the same as "testnotfounddir" string?

Reply:
done